### PR TITLE
Add instruction to run DeepSeek logits comparison

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

Add instruction to run DS logits comparison for correctness. 
* Add instruction from existing test, https://github.com/AI-Hypercomputer/maxtext/blob/75c8ef6aa33bd5192f88b5c35786a624420866c3/end_to_end/tpu/deepseek/v2-16b/test_deepseek.sh#L34 
* Remove golden logits for DS v2

Related: b/417234725

# Tests

Test locally: https://paste.googleplex.com/6485256997437440

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run end-to-end tests tests and provided workload links above if applicable.
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: end_to_end/tpu/deepseek/Run_DeepSeek.md [hunk 1]
```diff
@@ -135,6 +135,47 @@ python3 -m MaxText.decode MaxText/configs/base.yml \
...
+    --model-id=deepseek-ai/DeepSeek-V2-Lite \
+    --output-path=golden_DeepSeek-V2-Lite.jsonl \
+    --prompts='I love to,Today is a,What is the,' \
+    --gcs_bucket=${LOCAL_DIRECTORY}
+```
+
+One example command to run logits comparison with V2-Lite on CPU.
+
+```sh
+python3 -m MaxText.tests.forward_pass_logit_checker \
```
Comment (bvandermoon) [inline end_to_end/tpu/deepseek/Run_DeepSeek.md:158]: Just for my understanding - this will also be run with every PR as part of the unit tests, right?

**Review by bvandermoon — APPROVED** [review]

File: end_to_end/tpu/deepseek/Run_DeepSeek.md (hunk 1 above)
Comment (RissyRan) [reply end_to_end/tpu/deepseek/Run_DeepSeek.md:158]: No, it's not in the unit test yet for this logit test, and same for other models.

File: end_to_end/tpu/deepseek/Run_DeepSeek.md [hunk 2]
```diff
@@ -135,6 +135,47 @@ python3 -m MaxText.decode MaxText/configs/base.yml \
...
+* **MMLU Score Validation**: We validate the MMLU score against established benchmarks.
+
+One example command to generate golden logits from HuggingFace for V2-Lite on CPU.
+
+```sh
+LOCAL_DIRECTORY='MaxText/test_assets/'
+python3 -m MaxText.scratch_code.generate_hf_golden_logits \
+    --model-id=deepseek-ai/DeepSeek-V2-Lite \
+    --output-path=golden_DeepSeek-V2-Lite.jsonl \
+    --prompts='I love to,Today is a,What is the,' \
```
[the remark this answers is not in the document]
Comment (RissyRan) [reply end_to_end/tpu/deepseek/Run_DeepSeek.md:151]: Unfortunately we haven't handled the comma inside of prompts yet. For training logics rest, those three prompts are commonly used for our models. But we could definitely increase the length in the future if needed with sentences.

**Review by shralex — APPROVED** [review]
